Argo/Types/Decoded.swift

```diff
@@ -85,3 +85,10 @@ public extension Decoded {
     }
   }
 }
+
+public func ?? <T>(lhs: Decoded<T>, rhs: T) -> T {
```

Comment:
Yeah, wasn't sure. I feel like we need to reorganize all of the operators into different files.